src/theme.js

```diff
@@ -1,243 +1,252 @@
+      "quickInput.foreground" : color.fg.default,
+
+      "statusBar.foreground"             : color.fg.muted,
+      "statusBar.background"             : color.canvas.default,
+      "statusBar.border"                 : color.border.default,
+      "statusBar.noFolderBackground"     : color.canvas.default,
+      "statusBar.debuggingBackground"    : color.danger.emphasis,
+      "statusBar.debuggingForeground"    : color.fg.onEmphasis,
+      "statusBarItem.prominentBackground": color.canvas.subtle,
+
+      "editorGroupHeader.tabsBackground": color.canvas.inset,
+      "editorGroupHeader.tabsBorder"    : color.border.default,
+      "editorGroup.border"              : color.border.default,
+
+      "tab.activeForeground"        : color.fg.default,
+      "tab.inactiveForeground"      : color.fg.muted,
+      "tab.inactiveBackground"      : color.canvas.inset,
+      "tab.activeBackground"        : color.canvas.default,
+      "tab.hoverBackground"         : color.canvas.default,
+      "tab.unfocusedHoverBackground": color.neutral.subtle,
+      "tab.border"                  : color.border.default,
+      "tab.unfocusedActiveBorderTop": color.border.default,
+      "tab.activeBorder"            : color.canvas.default,
+      "tab.unfocusedActiveBorder"   : color.canvas.default,
+      "tab.activeBorderTop"         : color.primer.border.active,
+
+      "breadcrumb.foreground"               : color.fg.muted,
+      "breadcrumb.focusForeground"          : color.fg.default,
+      "breadcrumb.activeSelectionForeground": color.fg.muted,
+      "breadcrumbPicker.background"         : color.canvas.overlay,
+
+      "editor.foreground"                 : color.fg.default,
+      "editor.background"                 : color.canvas.default,
+      "editorWidget.background"           : color.canvas.overlay,
+      "editor.foldBackground"             : color.neutral.muted,
+      "editor.lineHighlightBackground"    : color.codemirror.activelineBg,
+      "editor.lineHighlightBorder"        : onlyDarkHighContrast(color.accent.fg),
+      "editorLineNumber.foreground"       : color.codemirror.linenumberText,
+      "editorLineNumber.activeForeground" : color.fg.default,
+      "editorIndentGuide.background"      : color.border.muted,
+      "editorIndentGuide.activeBackground": color.border.default,
+      "editorWhitespace.foreground"       : color.fg.subtle,
+      "editorCursor.foreground"           : color.accent.fg,
+
+      "editor.findMatchBackground"            : color.attention.muted,
+      "editor.findMatchHighlightBackground"   : color.attention.subtle,
+      "editor.linkedEditingBackground"        : color.accent.subtle,
+      "editor.inactiveSelectionBackground"    : color.accent.subtle,
+      "editor.selectionBackground"            : color.accent.muted,
+      "editor.selectionHighlightBackground"   : color.success.muted,
+      "editor.selectionHighlightBorder"       : null, // I need to figure this out as I think this was nothing....
```

Comment:
@simurai Do you know what's happening here? I think those were just transparent.

Reply:
Hmm.. right. These seem to be `transparent`. Maybe they are just a left over from playing around and then not needing them or intentionally setting them to transparent? I guess it could also just be `#00000000`. Will `null` render as transparent too?